packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -198,8 +198,8 @@ const MenuGroup = (props: MenuGroupProps) => {
     props.level === 2 && props.isActive && !props.isExpanded;
   return (
     <ul
-      id={`${props.id}-group`}
```

Comment:
Apparently querying for this ID (in tests) randomly fails if the value starts with a number (when using UUIDs).

<img width="755" alt="image" src="https://user-images.githubusercontent.com/1110551/185348661-7a8541f0-e952-4043-8204-583bd50fadc0.png">

Therefore I changed the prefix to be `group-`

Reply:
😮